pipelines/test/ocp/aws/pipeline.yaml

```diff
@@ -0,0 +1,218 @@
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-ocp-aws
+spec:
+  params:
+    # Cluster related parameters
+    - name: cluster-name
+      type: string
+    - name: region
+      type: string
+      default: "us-east-1"
+    - name: master-size
+      type: string
+      default: "m6g.xlarge"
+    - name: worker-size
+      type: string
+      default: "m6g.large"
+    - name: installer-arch
+      type: string
+      default: "arm64"
+    # Kuadrant/RHCL installation related parameters
+    - default: quay.io/kuadrant/kuadrant-operator-catalog:v1.2.0
+      description: Kuadrant/RHCL index image, leave empty to install current RHCL GA
+      name: index-image
+      type: string
+    - default: stable
+      description: Kuadrant channel (stable, preview)
+      name: channel
+      type: string
+    - default: kuadrant-operator
+      description: Operator name (kuadrant-operator, rhcl-operator)
+      name: operator-name
+      type: string
+    - default: ocp
```

Comment:
I think we should default to `ossm3` now - the mTLS is not supported if using `ocp` so all our mTLS tests would fail. For a context see https://github.com/Kuadrant/testsuite-pipelines/pull/144